lib/Conch/Controller/Schema.pm

```diff
@@ -46,6 +46,22 @@ sub get ($c) {
 	}
 
 	my $schema = $validator->get("/definitions/$name");
+
+    my sub inline_ref ( $ref, $schema ) {
```

Comment:
If I understand correctly, this is inlining the underlying definitions whenever there is a `$ref`? Could this be shown in a test?  I guess something like `$device_report_response->json_is('/system_uuid' => { type => 'string', pattern => '...' })`

Reply:
I'll make it explicit. It's implicit in the test for a device report but yeah I'll make an explicit test for it.

Reply:
Is it? I don't see where that's tested.

Reply:
The test for the device report failed before I copied that code over from `misc/extract-schema` :)

I've added an explicit test now in the earlier test for the Login message to see if the `$ref` we have there is inlined.